webapp/packages/plugin-data-spreadsheet-new/src/theme/styles.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import type { ThemeSelector } from '@cloudbeaver/core-theming';
+
+import type Dark from './dark.module.scss';
+import type Light from './light.module.scss';
+
+export const DATA_GRID_BASE_STYLES: ThemeSelector = async theme => {
+  let styles: typeof Light & typeof Dark;
+  const dark = (await import('./dark.module.scss')).default;
+  const light = (await import('./light.module.scss')).default;
```

Comment:
had preloaded all styles at one time, cause the theme did not apply until `<TableGrid>` component was fully rerendered (reload page or tab)